subprojects/core-api/src/main/java/org/gradle/api/internal/Generated.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal;
```

Comment:
❓ This annotation ends up on public members, I think it should be part of the public api

Reply:
That was my biggest question, I left it as internal because I don't expect anyone to use it on their project.
`org.gradle.api.Generated` then?

Reply:
Yep, sounds good